test/onnx/test_onnxscript_no_runtime.py

```diff
@@ -0,0 +1,164 @@
+
+        torch.onnx.register_custom_op_symbolic(
+            symbolic_name="aten::selu",
+            symbolic_fn=custom_selu,
+            opset_version=self.opset_version,
+        )
+
+        # 2. Register layer_norm onnxscript function as custom Op
+        @onnxscript.script(custom_opset)
+        def layer_norm(
+            X, axes: List[int], weight: FLOAT[...], bias: FLOAT[...], eps: float
+        ):
+            mean = op.ReduceMean(X, axes=axes)
+            D = X - mean  # op.Sub(X, mean)
+            DD = D * D  # op.Mul(D, D)
+            var = op.ReduceMean(DD, axes=axes)
+            vareps = var + eps  # op.Add(var, eps)
+            stddev = op.Sqrt(vareps)
+            invstddev = op.Reciprocal(stddev)
+            normalized = D * invstddev  # op.Mul(D, invstddev)
+            normalizedw = op.CastLike(
+                normalized, weight
+            )  # Type issue if missing this Op
+            normalizedscaled = normalizedw * weight  # op.Mul(normalized, weight)
+            return normalizedscaled + bias
+
+        @torch.onnx.symbolic_helper.parse_args("v", "is", "v", "v", "f", "none")
+        def custom_layer_norm(
+            g, input, normalized_shape, weight, bias, eps, cudnn_enable
+        ):
+            # TODO: move the comprehension into local function once
+            # it's supported by onnxscript
+            axes = [-i for i in range(len(normalized_shape), 0, -1)]
+            return g.onnxscript_op(
+                layer_norm, input, weight, bias, axes_i=axes, eps_f=eps
+            ).setType(input.type())
+
+        torch.onnx.register_custom_op_symbolic(
+            symbolic_name="aten::layer_norm",
+            symbolic_fn=custom_layer_norm,
+            opset_version=self.opset_version,
+        )
+
+        # 3. export two models
+        x = torch.randn(1, 2, 3, 4, requires_grad=True)
+        model_selu = torch.nn.SELU()
+        selu_onnx = io.BytesIO()
+        torch.onnx.export(model_selu, x, selu_onnx, opset_version=self.opset_version)
+
+        N, C = 3, 4
+        y = torch.randn(N, C)
+        model_layer_norm = torch.nn.LayerNorm(C)
+        layer_norm_onnx = io.BytesIO()
+        torch.onnx.export(
+            model_layer_norm, y, layer_norm_onnx, opset_version=self.opset_version
+        )
+
+        # 4. test on models
+        selu_proto = onnx.load(io.BytesIO(selu_onnx.getvalue()))
+        layer_norm_proto = onnx.load(io.BytesIO(layer_norm_onnx.getvalue()))
+
+        self.assertEqual(len(selu_proto.functions), 1)
+        self.assertEqual(len(layer_norm_proto.functions), 1)
+        self.assertEqual(selu_proto.functions[0].name, "Selu")
+        self.assertEqual(layer_norm_proto.functions[0].name, "layer_norm")
+
+    def test_loop_registration(self):
+        # Control flow is tested for _find_onnxscript_op function in torch/onnx/utils.py,
+        # which has recursive logic to go through every nodes with subgraph in model proto
```

Comment:
@justinchuby Added comments. LMK you have any suggestion for this comment.

Reply:
looking good. thanks a lot!